gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpec.java

```diff
@@ -358,4 +361,50 @@ public URI getUri() {
   public Boolean isExplain() {
     return ConfigUtils.getBoolean(getConfig(), ConfigurationKeys.FLOW_EXPLAIN_KEY, false);
   }
+
+  @Slf4j
+  public static class Utils {
```

Comment:
Any advantage defining a nested Utils class instead of exposing the utility methods directly as static methods in FlowSpec class?

Reply:
a separate util class is preferred over util methods, to keep util methods cleanly separate.
a nested util class instead of an outer class is just a coding style preference.